src/profile_service/index.js

```diff
@@ -1,11 +1,9 @@
 'use strict'
 
-const crypto = require('crypto');
+const yotiRequest = require('../yoti_request');
+const RequestPayload = require('../request_payload');
```

Comment:
Shouldn't we use `-` instead of `_` for name separator? To keep it consistent with `proto-root`